terminus-ui/popover/src/popover.component.md

```diff
@@ -0,0 +1,95 @@
+<h1>Popover</h1>
+
+Popover component is designed to pop up content based on user trigger.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic info](#basic-info)
+- [Basic Usage](#basic-usage)
+  - [Position](#position)
+  - [Hide on Blur](#hide-on-blur)
+  - [Open on load](#open-on-load)
+- [Events](#events)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Basic info
+
+We wrap our popover trigger/component with a third party library, named [popper.js](https://github.com/popperjs/popper.js).
+`popper.js` is not a required import for ui library but if it's not imported and consumers try to use popover component, an error will show up. 
```

Comment:
Good callout here. We should also add the install instructions and a link to the popper.js docs.

Also, this just reminded me, we should add Popper.js to the `ACKNOWLEDGEMENT.md` file